DMTR-53.tex

```diff
@@ -147,47 +147,297 @@ \subsection{Summary Table}
+		\begin{verbatim}  
+File "/software/lsstsw/stack3_20171023/stack/miniconda3-4.3.21-10a4fa6/
+Linux64/meas_algorithms/14.0-7-g23fdbe95+15/python/lsst/meas/algorithms/
+measureApCorr.py", line 251, in run
+    (name, len(subset2), self.config.minDegreesOfFreedom+1))
+RuntimeError: Unable to measure aperture correction for required algorithm 
+'base_PsfFlux': only 0 sources, but require at least 2.
+\end{verbatim}
+\end{itemize}
+
+We inspected the \texttt{DIAObject} table by opening the 
+\texttt{l1db/association.db} sqlite database produced by \texttt{ap\_pipe} 
+with \texttt{sqlite3}.  Issuing the query \texttt{select * from dia\_objects limit 10;} confirmed that the database was non-empty.
+
+
 \subsection{AG-00-10}
+
+\textit{Generated PVIs include science, mask, and variance images; background model; zeropoint; PSF, and WCS: PASSED}
+
+For all of the generated PVIs, we used Butler calls to check for the existence
+image, mask, and variance planes; zeropoints, PSFs, and WCS; and background models.
+
+All dataIds where data was generated in AG-00-05 had all of the required components.
+
+\textit{Masks and background are appropriately applied: PARTIALLY PASSED}
+
+We chose two visits and five ccds at random:
+visit 411673, ccds 15, 32, 45, 55, and 57, and
+visit 410915, ccds 3,  8, 23, 41, and 44.
+
+We used the Butler and DS9 to examine these PVIs.  We noted a few unmasked 
+bad columns as well as the effect of crosstalk; crosstalk correction for DECam
+was not enabled at the time of this processing.
+
 \subsection{AG-00-15}
+
+\textit{Difference images include provenance information relating to the 
+identity of the input exposures and the PSF matching kernel:  FAILED}
+
+No direct provenance information about the identity of the calexp and template
+exposures is available through current Butler interfaces.  While the input
+calexp can be reconstructed by matching dataIds from the processed image
+repository, determining the template applied requires a new call to
+\texttt{getCoaddAsTemplateTask}, with concomitant restrictions on 
+stack versioning, template respository association, etc.
+
+For each difference image, we used Butler calls to confirm that the PSF matching kernel was stored.  No failures were reported from the generated images.
+
+\textit{Masks are correctly propagated from the input image: PARTIALLY PASSED}
+
+We used the visits and ccds from AG-00-10:
+visit 411673, ccds 15, 32, 45, 55, and 57, and
+visit 410915, ccds 3,  8, 23, 41, and 44.
+
+We used the Butler and DS9 to compare the PVIs to the resulting differences. 
+The difference image masks were often modified relative to the PVIs, making 
+direct comparison more difficult.  In general the masks seemed to be propagated,
+although not all masks were--cosmic ray masks, for instance, did not appear 
+in the difference image.  It is unclear if this behavior is expected.
```

Comment:
I expect this is due to DM-9953 and hence is now fixed — probably worth stating as much.